### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ghost**
Sanitized code to protect against potential security issues.

### CODE REVIEW

**Comment by jim-p** [comment]
It's better, but not ideal. Lots of unnecessarily duplicated code. With some better structure that could be reduced significantly. Also there would be no need to have "inst-" and "install" in the filename. If you're hardcoding it, may as well reduce the redundancy.

**Comment by rbgarga** [comment]
In order to have patches accepted you must sign CLA as described at https://www.pfsense.org/about-pfsense/#legal

**Comment by gavit** [comment]
Please see attached,

Kind regards,
Gabor

File: config/openvpn-client-export/vpn_openvpn_export.php [hunk 1]
```diff
@@ -252,8 +252,20 @@
 		$exp_path = viscosity_openvpn_client_config_exporter($srvid, $usrid, $crtid, $useaddr, $verifyservercn, $randomlocalport, $usetoken, $password, $proxy, $openvpnmanager, $advancedoptions);
 	}
 
-	if (substr($act, 0, 4) == "inst") {
-		$exp_name = urlencode($exp_name."-install.exe");
+	if (substr($act, 0, 6) == "inst-x") {
+		if (substr($act, 6, 9) == "86-") {
+			if (substr($act, 9, 11) == "xp) {
```
Comment (phil-davis) [inline config/openvpn-client-export/vpn_openvpn_export.php]: Missing closing double-quotes.
Same syntax error in 4 places.

**Comment by rbgarga** [comment]
This repository is obsolete, it was used only for pfSense 2.2 and earlier versions.

For current pfSense versions, packages are hosted at https://github.com/pfsense/freebsd-ports

**Pull request closed without merging**